tests/test_create.py

```diff
@@ -1361,7 +1361,7 @@ def test_conda_run_1(self):
             use_restricted_unicode=False, name=str(uuid4())[:7]
         ) as prefix:
             output, error, rc = run_command(Commands.RUN, prefix, "echo", "hello")
-            assert output == "hello" + os.linesep
+            assert output == f"hello{os.linesep}\n"
```

Comment:
If os.linesep is `\n` are we looking for `\n\n` now

Reply:
not entirely sure why this change was necessary but the change was test driven:
- [Windows failure](https://github.com/conda/conda/actions/runs/4696058510/jobs/8325797700#step:9:498)
- [Linux failure](https://github.com/conda/conda/actions/runs/4696058510/jobs/8325800428#step:3:276)

Reply:
Question: is the bash shell introducing the `\n`